File: data-jpa/src/test/java/top/bettercode/summer/data/jpa/repository/StaticUserRepository.java
package top.bettercode.summer.data.jpa.repository;

import java.util.List;
import top.bettercode.summer.data.jpa.JpaExtRepository;
import top.bettercode.summer.data.jpa.domain.StaticUser;

public interface StaticUserRepository extends JpaExtRepository<StaticUser, Integer> {

  List<StaticUser> findByLastName(String lastName);
}

File: data-jpa/src/test/kotlin/top/bettercode/summer/data/jpa/repository/StaticUserRepository.kt
package top.bettercode.summer.data.jpa.repository

import top.bettercode.summer.data.jpa.JpaExtRepository
import top.bettercode.summer.data.jpa.domain.StaticUser

interface StaticUserRepository : JpaExtRepository<StaticUser?, Int?> {
    fun findByLastName(lastName: String?): List<StaticUser?>?
}